test/parallel/test-cluster-send-handle-twice.js

```diff
@@ -1,36 +1,36 @@
 'use strict';
 // Testing to send an handle twice to the parent process.
 
-var common = require('../common');
-var assert = require('assert');
-var cluster = require('cluster');
-var net = require('net');
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const net = require('net');
 
-var workers = {
+const workers = {
   toStart: 1
 };
 
 if (cluster.isMaster) {
-  for (var i = 0; i < workers.toStart; ++i) {
-    var worker = cluster.fork();
-    worker.on('exit', function(code, signal) {
-      assert.equal(code, 0, 'Worker exited with an error code');
-      assert(!signal, 'Worker exited by a signal');
-    });
+  for (let i = 0; i < workers.toStart; ++i) {
+    const worker = cluster.fork();
+    worker.on('exit', common.mustCall(function(code, signal) {
+      assert.strictEqual(code, 0, 'Worker exited with an error code');
+      assert.strictEqual(signal, null);
```

Comment:
You could have left the `'Worker exited by a signal'` as the third argument here.